### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by GuptaEesh**
>Added rule page for quizzes.

rule-page-quiz

### CODE REVIEW

File: quizes/quizInstructions.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    border-radius: var(--gutter-one);
+    display: flex;
+    flex-direction: column;
+    height: var(--card-large);
+    margin: auto;
+    width: var(--card-large);
+}
+.rules{
+    color: var(--white);
+    line-height: var(--gutter-two);
```
Comment (FarhanMobashir) [inline quizes/quizInstructions.css]: Instead of naming as gutter , name it in terms of spacing as gutter is not generic and maybe cause confusion later.

**Review by FarhanMobashir — CHANGES_REQUESTED** [review]
Change the name of css variable to a more generic one

File: quizes/quizInstructions.css (hunk 1 above)
Comment (GuptaEesh) [reply quizes/quizInstructions.css]: yeah, You are right. Done✅.

**Review by FarhanMobashir — APPROVED** [review]
All good ✅

**Review by AnuragSahu11 — APPROVED** [review]
Everything looks good, just a suggestion add a little padding or margin to the design quiz text its sticking to the border

**Pull request merged**